cal-itp/benefits
# Feat: add CalFresh option

Closes #1915 and #1918 

This PR adds the radio button option for "CalFresh Cardholders" to the eligibility index. The description contains a link that opens a help modal about CalFresh Cardholder eligibility.

### Screenshots
<img src="https://github.com/cal-itp/benefits/assets/25497886/d0c9a46a-f903-4e85-b73f-1434d741ae1d" width="450">

<img src="https://github.com/cal-itp/benefits/assets/25497886/9dc6c26a-4d3c-4a62-8831-71a189ab460d" width="450">

### To test verifier ordering

#### Steps for setting up to test
**_From existing database_**
```
./bin/init.sh
```
will run migrations, including a migration that sets the initial display order for your existing verifiers.

_**From scratch**_
```
./bin/reset_db.sh
./bin/init.sh
```
will recreate your database, run migrations, and load fixtures which define the values for `display_order` (since fixtures are loaded after migrations).

#### Steps for testing
Start your local app, and log into your local admin interface to edit the order of your `Eligibility verifiers` by drag-and-drop.

The radio buttons on the eligibility index should be in the order you specified.
